app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/CGTPropertyDisposalsConnector.scala

```diff
@@ -59,8 +63,8 @@ class CGTPropertyDisposalsConnectorImpl @Inject()(http: HttpClient, servicesConf
   ): EitherT[Future, Error, HttpResponse] =
    EitherT[Future, Error, HttpResponse](
     entity.fold(
-      t => http.get(s"$bprBaseUrl/sautr/${t.sautr.value}"),
-      i => http.post(s"$bprBaseUrl/nino/${i.nino.value}", Json.toJson(BprRequest(i.name.firstName, i.name.lastName, i.dateOfBirth.value)))
+      t => http.post(bprUrl(Left(t.sautr)), JsNull),
+      i => http.post(bprUrl(i.id), Json.toJson(BprRequest(i.name.firstName, i.name.lastName, i.dateOfBirth.map(_.value))))
```

Comment:
could these identifiers be made sufficiently descriptive to reflect their semantics

Reply:
what kind of thing were you thinking?

Reply:
t -> trust
i -> individual

Reply:
generally not a fan of single character identifiers

Reply:
oh right, yeah sure